ForumProject/users/views.py

```diff
@@ -60,6 +66,76 @@ def post(self, request, *args, **kwargs):
         return response
 
 
+class RoleSelectionView(APIView):
+    permission_classes = [IsAuthenticated]
+
+    def post(self, request):
```

Comment:
The POST method here is fine, but consider if PUT might be more appropriate if the operation is idempotent (i.e., the server state is the same no matter how many times the request is repeated with the same parameters)

Reply:
I leave this method for now, but later I will change it, because there is still no functionality for creating a relationship  UserRoleCompany during registration